src/main/java/uk/gov/hmcts/reform/migration/service/DataMigrationServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.migration.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+@Service
+public class DataMigrationServiceImpl implements DataMigrationService<Object> {
```

Comment:
Is there a reason we give it an `Object` for the type? Could it not be `Map<String, Object>`?